packages/app-extensions/src/rest/request.js

```diff
@@ -30,6 +30,19 @@ const extractBody = response => {
     })
 }
 
+const extractBlobBody = response => {
```

Comment:
since this is almost a copy of extractBody: i think it should be possible to move the code to one function an pass the function name (blob()/json()) as argument